sdk/src/utils/serde.rs

```diff
@@ -178,6 +178,35 @@ pub mod cow_boxed_slice_prefix {
     }
 }
 
+pub mod boxed_slice_prefix_serde {
```

Comment:
I think I would rename `boxed_slice_prefix` and `cow_boxed_slice_prefix` tbh. The name is not explicit enough as they deal with prefixed hex bytes, not all prefixed slices. It would free the name for your new one, which is more appropriate.

Reply:
Rename to what?

Reply:
Hmm idk, is `boxed_slice_prefix_hex_bytes` too much ?

Reply:
Think it's fine, not used in many places anyways